linode/resource_linode_object_storage_key.go

```diff
@@ -75,8 +75,6 @@ func resourceLinodeObjectStorageKeyRead(d *schema.ResourceData, meta interface{}
 
 	d.Set("label", objectStorageKey.Label)
 	d.Set("access_key", objectStorageKey.AccessKey)
-	d.Set("secret_key", objectStorageKey.SecretKey)
```

Comment:
We don't want to overwrite the actual key with `[REDACTED]` as returned by the API for GET requests.